### ISSUE TITLE
cannot be loaded as its extension is not registered in assetExts

### ISSUE THREAD

**Issue by JongGyuChoi** [highcharts/highcharts-react-native#56]
Error: 'node_modules/@highcharts/highcharts-react-native/highcharts-layout/css/styles.css' cannot be loaded as its extension is not registered in assetExts

why this error occurred?

**Comment by J-Zeitler** [comment]
I have a similar error. can add

```
resolver: {
            assetExts: [...assetExts, 'css'],
        },
```

to be able to load the `.css`-file, but I also get errors for:

```
'node_modules/@highcharts/highcharts-react-native/highcharts-files/highcharts.js' cannot be loaded as its extension is not registered in assetExts
```

and if I try to add that to `assetExts` the entire bundle seems to break.

**Comment by sebastianbochan** [comment]
Im closing the ticket as duplicate of https://github.com/highcharts/highcharts-react-native/issues/39

**Comment by anacnfalcao** [comment]
@J-Zeitler Were you able to resolve the issue without setting the useCDN and useSSL to true???

**Comment by J-Zeitler** [comment]
@anacnfalcao I actually made my own wrapper.  use one plain html-file with the highcharts js css etc that I need and then bootstrap it with my own webview. Something like this:

```js
const html = await fetch(Expo.Asset.fromModule(require('./highcharts.html')).uri)
                .then(f => f.text())
```

instead of the equivalent (I suppose):
https://github.com/highcharts/highcharts-react-native/blob/f188b1a1f8cba516c904d7435719ce28293ac03e/dist/src/HighchartsReactNative.js#L26-L27

It's a bit ugly but I got it to work with Expo that way without any special configuration.
